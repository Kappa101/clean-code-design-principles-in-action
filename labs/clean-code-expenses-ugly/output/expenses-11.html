<html><body>
<h1>Month January</h1>
<br/>Expenses: 4
 Food: 8.5</br>
 Train: 5.5</br>
 Taxi: 8.5</br>
<hr>
<b>Total: 22.5</b>
</html></body>
