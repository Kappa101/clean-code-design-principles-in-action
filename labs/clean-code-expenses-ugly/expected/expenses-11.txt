Month January

Expenses: 4

Food: 10
Taxi: 15
Train: 16

Total: 33